swarm/network/stream/delivery_test.go

```diff
@@ -41,62 +41,10 @@ import (
 	"github.com/ethereum/go-ethereum/swarm/testutil"
 )
 
-//Tests initializing a retrieve request
-func TestStreamerRetrieveRequest(t *testing.T) {
-	regOpts := &RegistryOptions{
-		Retrieval: RetrievalClientOnly,
-		Syncing:   SyncingDisabled,
-	}
-	tester, streamer, _, teardown, err := newStreamerTester(regOpts)
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer teardown()
-
-	node := tester.Nodes[0]
-
-	ctx := context.Background()
-	req := network.NewRequest(
-		storage.Address(hash0[:]),
-		true,
-		&sync.Map{},
-	)
-	streamer.delivery.RequestFromPeers(ctx, req)
-
-	stream := NewStream(swarmChunkServerStreamName, "", true)
```

Comment:
`SwarmChunkServer` doesn't exist anymore, and `RETRIEVE_REQUEST` doesn't exist anymore.

Also this is a huge test, which only checks that if we issue a request to `RequestFromPeers`, we get specific messages back - `SubscribeMsg` for the `RETRIEVE_REQUEST` stream, and a `RetrieveRequestMsg` - this is rather useless for two reasons:
1. We are locking behaviour that doesn't provide much value.
2. We are testing a very basic functionality, which is hit 1000 times per a single integration test.

No need to lock this behaviour in such a complicated way. Not to mention that `RETRIEVE_REQUEST` stream serves no purpose in any production code for a long time.